pkg/helm/release/manager.go

```diff
@@ -180,9 +179,10 @@ func (m manager) InstallRelease(ctx context.Context) (*rpb.Release, error) {
 }
 
 // UpdateRelease performs a Helm release update.
-func (m manager) UpdateRelease(ctx context.Context) (*rpb.Release, *rpb.Release, error) {
+func (m manager) UpdateRelease(ctx context.Context, force bool) (*rpb.Release, *rpb.Release, error) {
```

Comment:
To future-proof this for other possible options, I'd suggest we use functional options instead of adding an explicit `force` parameter to the signature. WDYT?

And we could preemptively update the Install and Uninstall signatures as well.

```go
type UpdateOption func(*action.Upgrade) error

func ForceUpdate(force bool) UpdateOption {
    return func(u *action.Upgrade) error {
        u.Force = force
        return nil
    }
}

func (m manager) UpdateRelease(ctx context.Context, opts ...UpdateOption)  (*rpb.Release, *rpb.Release, error) {
    upgrade := action.NewUpgrade(m.actionConfig)
    upgrade.Namespace = m.namespace
    for _, o := range opts {
        if err := o(u); err != nil { 
            return fmt.Errorf("failed to apply upgrade option: %w", err)
        }
    }

    ...
}
```